subsys/bluetooth/host/scan.c

```diff
@@ -1557,7 +1557,26 @@ int bt_le_per_adv_sync_delete(struct bt_le_per_adv_sync *per_adv_sync)
 		err = bt_le_per_adv_sync_terminate(per_adv_sync);
 
 		if (!err) {
+			const struct bt_le_per_adv_sync_term_info term_info = {
```

Comment:
does it make sense to factor that out into a 'delete-with-cb' function (cf L917).

Reply:
Did consider this myself; I'll add a function for it :)